Common/cpp/Tools/UIRuntimeDecorator.cpp

```diff
@@ -59,6 +60,22 @@ void UIRuntimeDecorator::decorate(
   jsi_utils::installJsiFunction(uiRuntime, "_setGestureState", setGestureState);
   jsi_utils::installJsiFunction(
       uiRuntime, "_maybeFlushUIUpdatesQueue", maybeFlushUIUpdatesQueue);
+  jsi_utils::installJsiFunction(
+      uiRuntime,
+      "_obtainProp",
+#ifdef RCT_NEW_ARCH_ENABLED
+      [obtainPropFunction](
+          jsi::Runtime &rt,
+          const jsi::Value &shadowNodeWrapper,
+          const jsi::Value &propName) {
+        return obtainPropFunction(rt, shadowNodeWrapper, propName.asString(rt));
```

Comment:
Here we're simply converting from `jsi::Value` to `jsi::String`, while in the `obtainProp` function, there's a conversion from `jsi::String` to `std::string` 🫣

Reply:
🤯

Reply:
Then you should fix function signature in `PlatformDepMethodsHolder.h` and `NativeReanimatedModule.cpp` since it has `jsi::Value` there instead of `jsi::String`.